File: solve.py
import os
import re

html_list = open('CMS-webpage/wireframe_site/content-list.html', 'r', encoding='utf-8').read()
html_list = re.sub(r'location\.href=\'content-editor.html\?id=101\'', r'location.href=\'content-detail.html?id=101\'', html_list)
html_list = re.sub(r'location\.href=\"content-editor.html\?id=101\"', r'location.href=\"content-detail.html?id=101\"', html_list)

with open('CMS-webpage/wireframe_site/content-list.html', 'w', encoding='utf-8') as f:
    f.write(html_list)

html_det = open('CMS-webpage/wireframe_site/content-detail.html', 'r', encoding='utf-8').read()
print("Updated content-list to point to content-detail.html!")

File: solve.js
const fs = require('fs');
let list = fs.readFileSync('CMS-webpage/wireframe_site/content-list.html', 'utf8');
let editor = fs.readFileSync('CMS-webpage/wireframe_site/content-editor.html', 'utf8');
let detail = fs.readFileSync('CMS-webpage/wireframe_site/content-detail.html', 'utf8');

console.log('--- OUTPUT START ---');
console.log('List links to Editor?', list.includes('content-editor.html'));
console.log('List links to Detail?', list.includes('content-detail.html'));
console.log('Editor includes name?', editor.includes('name'));
console.log('Detail includes content_nm (kr)?', detail.includes('콘텐츠 이름') || detail.includes('콘텐츠명'));
console.log('Editor h1:', editor.match(/<h1.*?>.*?<\/h1>/)?.[0]);
console.log('Detail h1:', detail.match(/<h1.*?>.*?<\/h1>/)?.[0]);
console.log('--- OUTPUT END ---');
